spec/api/v1/external_users/claim_spec.rb

```diff
@@ -16,6 +16,9 @@
     /api/external_users/claims/advocates/supplementary
     /api/external_users/claims/advocates/supplementary/validate
 
+    /api/external_users/claims/advocates/hardship
```

Comment:
👍 yes this is good for AGFS 
see LGFS equivalent https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3341/files/d71bea25dc2edb4144f9e7fd772c502d1bb515fc#r420650769